force-app/main/default/classes/IBMSpeechToTextV1.cls

```diff
@@ -248,72 +254,104 @@ public class IBMSpeechToTextV1 extends IBMWatsonService {
     if (recognizeOptions.redaction() != null) {
       builder.query('redaction', String.valueOf(recognizeOptions.redaction()));
     }
+    if (recognizeOptions.processingMetrics() != null) {
```

Comment:
We want to remove `processingMetrics` and `processingMetricsInterval` from sync recognize since this returns multiple json objects. Keep `audioMetrics` though - that works fine.